es.md

```diff
@@ -0,0 +1,209 @@
+---
+layout: page
+title: Solidaridad para lxs Trabajadorxs durante la Pandemia del COVID-19
```

Comment:
no issues just want to ask what the politics / circulation of lxs and 'x' is in spanish-speaking parts of the world. In conversation with folks I'm used to it being much more prevalent in Puerto Rico, US, than even MX and very specific in Latin America. Are the connotations or any aspects we want to keep in mind around this?

(e.g., on it's ealry use in online, queer communities is what I'm most familiar with https://s3.amazonaws.com/academia.edu.documents/53992477/Onis_2017_Chiricu.pdf?response-content-disposition=inline%3B%20filename%3DWhat_s_in_an_x_An_Exchange_about_the_Pol.pdf&X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=ASIATUSBJ6BAOHXYQJCC%2F20200404%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20200404T175149Z&X-Amz-Expires=3600&X-Amz-Security-Token=IQoJb3JpZ2luX2VjEHEaCXVzLWVhc3QtMSJHMEUCIQClto%2F4n4dVbENwLWQv90V5saohXg6MxmARR8%2FzsqiY%2FwIgDFANGPD%2FWQTK4crCEYOmrMDxgLAwpKfgTMbzn4NVgdcqtAMIeRAAGgwyNTAzMTg4MTEyMDAiDNAHtLjNJTVORcXLqiqRA3uLelPlQ6x1wiXuZakSdCpo5VkaR%2Fzu89kC2mdV%2Bgs3Ij7iDg5YpvaozLutGavQ34GSFk%2F%2BouOKuGp8aKU5kF8KTRy26STZHpl%2F26zAt%2FrXyzo1%2F5eIzThQ7No0DZYCOPYo%2FqXXXuS%2F5BGJG7oaeH8PpeCiEjUVRcClXjjNTGaDWFpaGzLELXQptWedwaDpfoVw%2FEKZYit0dqhfWvnFHmTFAsANg7Oc5ntBjSOXttZ%2BicR3dlp847IxxiZDmfmjys7txB7Tk%2FPUsIaa4CefE15QJYshBjtVPPxy9UMPG4k9BrZjfx3cpnfwkM%2BcRE1Oq9U4IjGaGMSJ4aRmkmHlQQC59fAtvJRCct9BazxKcPdS9MjiWahIXURww72tE%2BNGhhLOB%2BV7ZdUc6o3YbLfLoniYCkAcwxxTNgfhk2nIuFWB0Oz9egNI83Mz42NG4jw%2F3AweTCZkZDFhp1ypJpA%2Bg5CuZR5PUPfzX%2BppYNLdUKSKLLNrWt1ncrmLED37gX%2B1JLJo7Q%2FPoO7St%2FKBuipsSuMsMPjmovQFOusB0o39orB4R8s9TQFj39nmg3y3isq1AmEcp9xjuLP8xgPiSKVbftCWiQgOV4JxUWu9FKMHciH%2FA2oRSxT%2FnX48i%2Be4Z9FBchniOWraHKSFDd2DrNDViYuL0q4iBtCpuIzs4vEmS8Hze3LzlP%2BxudYVwTcI2xor6xhpyCPv6dCN6ynV3wc1iptaYzzbaWgPb%2BUmKZPET%2BcNbcCAnsampbITGPt3Q6zxL%2B5GQzNPMrAsIBH5uKLgs%2BXrvnwacD%2FRSAHEFjuU9ZBaWu1l9g8RtAFMFinT5VWxabYpMQ77yIcTvEnaETUBZk4cK9oOow%3D%3D&X-Amz-SignedHeaders=host&X-Amz-Signature=cf5295fca53f48641ffc49d9e4bfaeb21920126ffe85fbadc1cdb7faf93b4002)

Reply:
I cannot access the link

Reply:
It's okay! the link isn't important, just a way to show how people talk abt in US. I am just wondering how lxs etc... is used elsewhere